config/core/resources/sequence.yaml

```diff
@@ -143,8 +143,6 @@ spec:
             properties:
               address:
                 type: object
-                required:
-                  - url
```

Comment:
That one _is_ required to satisfy the `Addressable` duck type, and that's OK because it only applies if `address` if set.

In other words, this prevents an object from being marshaled with `address: {}` and no address inside.

Reply:
Other resources ([broker](https://github.com/knative/eventing/blob/release-0.26/config/core/resources/broker.yaml#L100-L105), [imc](https://github.com/knative/eventing/blob/release-0.26/config/channels/in-memory-channel/resources/in-memory-channel.yaml#L141-L145), [parallel](https://github.com/knative/eventing/blob/release-0.26/config/core/resources/parallel.yaml#L163-L167)) have this requirement removed at some point probably because there was the same issue - when, for any reason, entry point URL is not available, the controller cannot update sequence's status:

```
26s         Warning   UpdateFailed                          sequence/foo                     Failed to update status for "foo": Sequence.flows.knative.dev "foo" is invalid: status.address.url: Required value
```

Reply:
Sounds like a nasty workaround to an actual bug to me, but since it's how it's done everywhere...